app/controllers/race_entries_controller.rb

```diff
@@ -1,45 +1,25 @@
 class RaceEntriesController < ApplicationController
-  def index
-    sort = params[:sort] || 'racers.last_name'
-    @race_entries = RaceEntry.includes(:racer, :race_edition).order(sort).map { |re| RaceEntryPresenter.new(re) }
-  end
-  
   def show
     @race_entry = RaceEntryPresenter.new(RaceEntry.find(params[:id]))
   end
   
-  def new
```

Comment:
This is not used in practice. There was a link to this action, but it does not work because the form does not allow a racer to be created with the race entry.

Instead, racers and race entries are created using the `RaceEditionsController#enter` endpoint.